.travis.yml

```diff
@@ -3,6 +3,7 @@ python: "3.5"
 dist: trusty
 sudo: true
 before_install:
+  - wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
```

Comment:
Please revert this unrelated change. By rebasing your branch on top of the current state of the target branch CI should pass out-of-the-box.